packages/next/src/server/dev/hot-reloader.ts

```diff
@@ -327,10 +328,19 @@ export default class HotReloader {
     return { finished }
   }
 
+  public setServerHmrError(error: Error | null): void {
+    this.hmrServerError = error
+  }
+
+  public clearServerHmrError(): void {
+    this.setServerHmrError(null)
+    this.send('reloadPage')
```

Comment:
We only want to trigger a reload if there were errors active on the client right?